sdk/daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/ledger/BlindingTransaction.scala

```diff
@@ -79,13 +102,17 @@ object BlindingTransaction {
             // fetch & exercise nodes cause divulgence
 
             case fetch: Node.Fetch =>
```

Comment:
What's the reason for not adding anything to `visibility` for create / LBK nodes?

Reply:
Because it is not needed by the engine, however, it is needed for model conformance as we include resolved key lookups as view input contracts => adding to visibility.